src/main/java/views/blackjack/blackjack.flt.html

```diff
@@ -58,6 +58,21 @@
 function display(game) {
 console.log(game);
 
+$("#player_cards").html("");
+for(i=0; i < game.user.hand.length; i++) {
+    card = game.user.hand[i];
+    cimg = "<img src='/assets/cards/" + card.value + card.suit.charAt(0).toLowerCase() + ".png'>";
```

Comment:
I don't believe so. The actual object as shown by the dev tools in chromium, show that the entire deck is ace of diamonds. Or more accurately 14 of Diamonds.